Client/src/Pages/Monitors/CreateNewMonitor.jsx

```diff
@@ -162,9 +185,15 @@ const CreateNewMonitor = () => {
               checked={selectedMonitorType === "port"}
               onChange={() => handleMonitorTypeChange("port")}
             />
-            <div className="monitors-gaps-medium"></div>
+            <div className="monitors-gaps-small-plus"></div>
             <div className="monitors-dropdown-holder">
-              <FlexibileTextField placeholder="Select a port to check" />
+              <Dropdown
+                id="ports-dropdown"
+                label="Select a port to check"
+                onChange={handlePortOptionChange}
+                options={portOptions}
```

Comment:
should this just be port instead of port options? Maybe that will fix the issue?

Reply:
No, apparently this is an array type that needs to be passed.